include/swift/AST/DiagnosticsSema.def

```diff
@@ -1064,7 +1064,9 @@ NOTE(note_call_to_initializer,none,
 NOTE(note_init_parameter,none,
      "in initialization of parameter %0", (Identifier))
 
-
+ERROR(missing_explicit_call_callable_value, none,
+      "value of callable type %0 has a 'callAsFunction' declaration that produces expected type %1; "
```

Comment:
How about just...

```suggestion
      "value of callable type %0 can be called to produce expected type %1; "
```

Reply:
That's better, just fixed :)
Thanks @xwu!